au4/src/projectscene/view/clipsview/trackslistclipsmodel.cpp

```diff
@@ -28,13 +30,27 @@ void TracksListClipsModel::load()
 
     m_trackList = prj->trackList();
 
-    //! TODO Subscribe on tracks changed
-
     m_dataSelectedTracks = selectionController()->dataSelectedOnTracks();
     selectionController()->dataSelectedOnTracksChanged().onReceive(this, [this](const std::vector<trackedit::TrackId>& tracks) {
         setDataSelectedTracks(tracks);
     });
 
+    m_trackList.onItemAdded(m_tracksNotifyReceiver.get(), [this](const trackedit::Track& track) {
```

Comment:
Why did you decide to use a separate object instead of just passing this?

Reply:
oh, I did it the same way as it is done in TracksListModel -> should I fix that in both?

Reply:
I asked Elnur... in general, he took this solution from MuseScore, it made sense there, but here and in TracksListModel it doesn't make sense :) You can just pass this :)

Reply:
fixed